deployment_report/model/viya_deployment_report.py

```diff
@@ -107,8 +107,22 @@ def gather_details(self, kubectl: KubectlInterface,
                 sas_custom_resources[kind] = details
 
         # create dictionary to store gathered resources #
-        gathered_resources: Dict = dict()
+        gr: Dict = dict()
```

Comment:
changing to a shorter name reduces the readability of the code, I think this should be put back to the longer name it had before

Reply:
- replaced variable gr to gathered_resources for readability